.credo.exs

```diff
@@ -37,6 +37,7 @@
           ~r"/src/metrics",
           ~r"/src/minerl",
           ~r"/src/parse_trans",
+          ~r"/src/stream_data",
```

Comment:
I think we might need to add this to our .gitignore

Reply:
fixed in https://github.com/apache/couchdb/pull/2662/commits/edd533eba85302eba37fabfe6c36b3731a24ed15